src/Logging/Logging.Abstractions/src/NullExternalScopeProvider.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+
+namespace Microsoft.Extensions.Logging.Abstractions
```

Comment:
We generally avoid putting `Abstractions` in a namespace. Just `Microsoft.Extensions.Logging`